app/presenters/prior_authority/check_answers/compact_further_information_card.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module PriorAuthority
+  module CheckAnswers
+    class CompactFurtherInformationCard < Base
+      attr_reader :further_information
+
+      def initialize(further_information)
+        @group = 'about_request'
+        @section = 'compact_further_information'
+        @further_information = further_information
+        super()
+      end
+
+      def title
+        super(date: further_information.created_at.to_fs(:stamp))
+      end
+
+      def row_data
+        base_rows
+      end
+
+      def base_rows
+        [
+          {
+            head_key: 'information_request',
+            text: simple_format(further_information.information_requested),
+          },
+          {
+            head_key: 'your_response',
+            text: supporting_documents,
+          },
+        ]
+      end
+
+      private
+
+      def supporting_documents
+        links = further_information.supporting_documents.map do |document|
+          govuk_link_to(document.file_name,
+                        Rails.application.routes.url_helpers.prior_authority_download_path(document))
+        end
+        response = further_information.information_supplied
+        parts = ([response, tag.br] + links.flat_map { [_1, tag.br] }[..-2])
```

Comment:
```
parts = [response] + links.flat_map { [tag.br, _1] }
```

Reply:
THANK YOU! For the life of me I couldn't see how to make this less ugly.